55.md

```diff
@@ -0,0 +1,346 @@
+   </td>
+  </tr>
+  <tr>
+   <td>Dimension 
+   </td>
+   <td>akin to the tags in the example given in the earlier proposals
+   </td>
+   <td>Tag ‘y’
+   </td>
+  </tr>
+  <tr>
+   <td>Category
+   </td>
+   <td>Used to categorise Dimensions and allow re-use of Dimensions in different contexts and thus differentiate different uses of a given Dimension. 
+<p>
+For example, if someone is deemed a good “judge” of others in the field of, say, Austrian economics, that same good judgement may not apply in the field of, say, orchid gardening. Category allows for ratings to be filtered (by clients or relays) for more than one purpose and allows for applications to more flexibly share and reuse ratings. 
+   </td>
+   <td>Tag ‘x’
+   </td>
+  </tr>
+  <tr>
+   <td>Scale
+   </td>
+   <td>Could be -1 to 1, say, and thus support positive and negative ratings - similar to the earlier proposals.
+   </td>
+   <td>Tag ‘scale’
+   </td>
+  </tr>
+  <tr>
+   <td>Comment
+<p>
+(optional)
+   </td>
+   <td>Used to optionally provide supporting text.  
+   </td>
+   <td>event.content
+   </td>
+  </tr>
+  <tr>
+   <td>Expiration timestamp 
+<p>
+(optional)  
+   </td>
+   <td>This allows reputation events to be used as time-limited accreditations. 
+   </td>
+   <td>Tag ‘expiration’
+   </td>
+  </tr>
+  <tr>
+   <td>RatingProofType
+   </td>
+   <td>Indicates the method by which proof of the rating can be achieved. For example, ‘Type 2 = check that external rater has contributed to a Github project.’ Inextricably related to the Nostr rating’s Dimension.
+<p>
+Type 1 = internal Nostr rating.
+   </td>
+   <td>Tag ‘rateprooftype’
+   </td>
+  </tr>
+  <tr>
+   <td>ExternalRatingProof
+   </td>
+   <td>(conditionally required, depending on Rating Proof Type)
+<p>
+This field is required for any external rating. May be a URL. Will be described more fully in an upcoming NIP to allow external proofs. 
+   </td>
+   <td>Tag ‘exrateproof’
+   </td>
+  </tr>
+  <tr>
+   <td>Informant
+   </td>
+   <td>Given this scheme supports Nostr recognising external ratings, the informant will be the Nostr pubkey who creates the Nostr record. This is not necessarily the rater. 
+   </td>
+   <td>event.pubkey
+   </td>
+  </tr>
+  <tr>
+   <td>Whitelisted/blacklisted IP address
+   </td>
+   <td>
+   </td>
+   <td>Tag ‘p’
+   </td>
+  </tr>
+</table>
+
+
+
+
+
+Apps can decide the categories and dimensions they wish to use. There is no centrally-defined list of categories or dimensions. Other apps may adopt or query reputation events made by other apps if they are interesting to them. Clients may synthesise disparate reputations that will enable wholly new use cases to emerge. 
+
+Regarding blacklist/whitelist registration. It is recognised that blacklist/whitelist detection and management is a complex issue and outside the scope of this NIP. The whitelist/blacklist tag is just to bring such lists in-protocol, allow sharding and enable efficient selective replication. 
```

Comment:
NACK

Blacklists have no place in a "censorship resistant" protocol particularly based on reputation system defined in this NIP.

Reply:
Blacklists can and will be used by users and relays alike. Nostr's censorship resistance doesn't emerge out of everybody processing all messages ever. It emerges out of relays being redundant.

Users will want block lists and I want to know your block list to at least de-boost who you block, so having them public makes a lot of sense.

Not even the most wild-west relay on TOR will work without some kind of black- or white listing as storage/bandwidth is not free and infinite.

Reply:
> Blacklists can and will be used by users and relays alike. Nostr's censorship resistance doesn't emerge out of everybody processing all messages ever. It emerges out of relays being redundant.

No they wont be used. Dont span this repo with non sense.